src/Support/OMOptions.cpp

```diff
@@ -40,8 +40,3 @@ llvm::cl::opt<int> onnxOpTransformThreshold("onnx-op-transform-threshold",
 llvm::cl::opt<bool> onnxOpTransformReport("onnx-op-transform-report",
     llvm::cl::desc("Report diagnostic info for op transform passes."),
     llvm::cl::init(false), llvm::cl::cat(OMPassOptions));
-
```

Comment:
Removed because we should add the `-maccel` option where all the other external options are (like -O3, -mcpu, etc..). I added it there.

Reply:
agreed